client/lib/abtest/active-tests.js

```diff
@@ -127,4 +127,13 @@ export default {
 		allowExistingUsers: true,
 		localeTargets: 'any',
 	},
+	checklistThankYouPage: {
```

Comment:
I don't think it's possible because users will purchase the paid plan after the A/B test is activated. How do we know what A/B test is supposed to be activated at the first step of the signup flow?